### ISSUE TITLE
flutter/website setup instructions need some love in gLinux (at least)

### ISSUE THREAD

**Issue by ditman** [flutter/website#5970]
_TL;DR: This project is slightly hard to set up in a fresh gLinux instance_

When following the instructions to run/deploy flutter/website as described in the README, I ended up with a problem:

```
./tool/serve.sh: line 68: bundle: command not found
```

I needed to install a specific version of bundler for this to work:

```
sudo gem install bundler:1.17.3
```

(I thought this was taken care of by some of the pre-install/install scripts?)

It then proceeded to complain about `jekyll` not being installed, and asked me to do a `gem install`.

After installing `jekyll`, I get a ruby-sass error:

```
           Assets:  Undefined mixin 'float-right'.
         Asset File css/_toc_shared.scss:54:in `float-right'
  Liquid Exception: Sass Error in /_layouts/base.html
jekyll 3.9.1 | Error:  Sass Error
```

My problem now is that I don't know if my "system ruby" is full of garbage, or the gems are in the right spot. The installation steps would benefit from some love in Linux (at least gLinux). I also can't bring this back to how it was *before* I started typing commands to get `nvm` and `rvm` going 😟

**Comment by parlough** [comment]
I'm going to close this as the site infrastructure is completely different, based on Docker after https://github.com/flutter/website/commit/e5954300435164c41fdc4e7a935fdca2f20caebc, which should solve some of these concerns.
